packages/appstore/src/components/main-title-bar/account-type-dropdown.tsx

```diff
@@ -5,6 +5,7 @@ import { Dropdown } from '@deriv/components';
 import { account_types } from 'Constants/platform-config';
 import { useStores } from 'Stores';
 import './account-type-dropdown.scss';
+import { Localize, localize } from '@deriv/translations';
```

Comment:
If we will use only the Localize component, then we can remove tthe other localize function import.

`import { Localize } from '@deriv/translations';`